### ISSUE TITLE
Author requirements clarification for aria-placeholder

### ISSUE THREAD

**Issue by feerrenrut** [w3c/aria#541]
The aria-placeholder section mentions:

> Authors should present this hint to the user by displaying the hint text at any time the control's value is the empty string. This includes cases where the control first receives focus, and when users remove a previously-entered value.

To me it seems unclear whether the author is required to remove the `aria-placeholder` attribute when a value is entered. Some guidance on how assistive technology might be expected to behave could also help to clarify this.

For instance, is it the responsibility of the AT to check that there is no value set on the field before reporting the placeholder text? Further to this, could there be some clarification of the use case for why aria-placeholder should be always available to AT even when it is not visible to sighted users?

**Comment by jnurthen** [comment]
Firstly - sorry this has taken so long... I'm trying to go through old untriaged issues.

It is up to the AT how to present information to users. The best approach seems to the the one you have taken in that placeholder (and aria-placeholder) would only be available to AT users when the field is empty. This would be the same as the visual experience.
I'm not sure what to change in the spec text to clarify this though as we generally don't give normative statements around AT behaviour.
